influxql/ast.go

```diff
@@ -2759,6 +2769,65 @@ func (s *DropShardStatement) RequiredPrivileges() (ExecutionPrivileges, error) {
 	return ExecutionPrivileges{{Admin: true, Name: "", Privilege: AllPrivileges}}, nil
 }
 
+// ShowSeriesCardinalityStatement represents a command for listing series cardinality.
+type ShowSeriesCardinalityStatement struct {
+	// Database to query. If blank, use the default database.
```

Comment:
Can the comment make clear that this is used to override the default database? At least that's what it appears to do.